files/ko/web/html/element/input/file/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: <input type="file">
-slug: Web/HTML/Element/Input/file
+slug: Web/HTML/Element/input/file
```

Comment:
@yin1999 Thanks for the status change!

 Just asking because I'm curious. In the slug, some are uppercase and some are lowercase. What difference does the uppercase letter determine?

Does it depend on the folder?

- `slug: Web/HTML/Element/input`
- `slug: Web/HTML/Element/video`
- `slug: Glossary/Firewall`
- `slug: Web/API/Notification`
- `slug: Web/API/window/requestAnimationFrame`

Reply:
I just made some changes in MDN/yari. So the MDN bot could also sync the slug in case sensitive mode. The slug case change made here is just from https://github.com/mdn/content/blob/main/files/en-us/web/html/element/input/file/index.md

I think it still important to keep the slug just the same as it in en-US documents(as the team that maintains en-US document may made some changes on this for any reasonable points) :)

Reply:
I understand. Thank you for the reply.

Have a good weekend :)